### PR TITLE
FIX-#16: Apply `black` to all project

### PR DESCRIPTION
**Pull request by prutskov**
PR contains changes after:
```bash
black .
```

from the root of `unidist`

Signed-off-by: Alexey Prutskov <alexey.prutskov@intel.com>

### CODE REVIEW

**Comment by YarShev** [comment]
Let's apply all linters we are going to use for formatting. Namely, black and flake8.

**Comment by YarShev** [comment]
After that we should set up initial CI for the linters as part of a separate issue that is a sub-task of #9. Would you be able to do it?

**Comment by prutskov** [comment]
Yes, I'll do this

File: unidist/core/backends/mpi/core/communication.py [hunk 1]
```diff
@@ -4,22 +4,26 @@
...
+import time
 import mpi4py
-mpi4py.rc(recv_mprobe=False)
 from mpi4py import MPI
 
-import time
-
 from unidist.core.backends.mpi.core.serialization import MPISerializer
 
+mpi4py.rc(recv_mprobe=False)
```
Comment (prutskov) [inline unidist/core/backends/mpi/core/communication.py:13]: @no-ponomarev please, take a look on this. Does this affect MPI execution? (I've tested on census - this works well)

Comment (no-ponomarev) [reply unidist/core/backends/mpi/core/communication.py:13]: This call should be done before any MPI initializations, therefore moving this call may affect corner cases (which I don't remember now) and should be undo.

Comment (prutskov) [reply unidist/core/backends/mpi/core/communication.py:10]: Ok. Ignoring comment was added for imports after `mpi4py.rc(recv_mprobe=False)` to make `flake8` check passed.

**Comment by prutskov** [comment]
@no-ponomarev, @YarShev `black` and `flake8` were applied. PR can be merged. CI checks will be included in a separate PR as part of #9 task

File: unidist/core/backends/mpi/core/communication.py (hunk 1 above)
Comment (no-ponomarev) [reply unidist/core/backends/mpi/core/communication.py:10]: Thanks! Not sure how to configure mpi4py outside this import code, maybe add a TODO to rework this later

Comment (prutskov) [reply unidist/core/backends/mpi/core/communication.py:11]: Agree with message. `TODO` comment was added

**Review by no-ponomarev — APPROVED** [review]
LGTM

**Review by YarShev — APPROVED** [review]
@prutskov, LGTM, thanks!

**Pull request merged**